app/PilotPassportEnrollment.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+
+class PilotPassportEnrollment extends Model {
+    protected $table = "pilot_passport_enrollment";
+
+    public function getNameAttribute() {
+        $pilot = RealopsPilot::find($this->cid);
```

Comment:
I think we should be able to do `$this->pilot()` here since the relationship is defined.

This should be able to be done everywhere with a relationship defined. I'm not gonna comment on all of them (and honestly I'll probably miss a few), but this is one of the benefits with relationships.

Reply:
Done